packages/components/src/components/icon/common/ic-email-verified.svg

```diff
@@ -0,0 +1 @@
+<svg width="120" height="120" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M6 36a6 6 0 0 0-6 6v72a6 6 0 0 0 6 6h84a6 6 0 0 0 6-6V42a6 6 0 0 0-6-6H6zm8.66 19.733a2.9 2.9 0 0 1 1.027-1.26c.45-.307.975-.471 1.512-.473h61.602a2.693 2.693 0 0 1 1.512.473 2.9 2.9 0 0 1 1.027 1.26L49.707 81.798A2.674 2.674 0 0 1 48 82.42c-.62 0-1.22-.219-1.707-.622L14.66 55.733zm-.26 7.356v29.764l17.961-14.97L14.4 63.09zm1.282 38.413c.451.31.978.477 1.517.481l61.602.017a2.715 2.715 0 0 0 1.517-.482c.452-.31.81-.75 1.03-1.268L59.082 81.665l-5.93 4.885a8.024 8.024 0 0 1-5.124 1.87 8.024 8.024 0 0 1-5.125-1.87l-5.93-4.885-22.32 18.569c.22.517.577.957 1.029 1.268zM81.6 92.845V63.088L63.639 77.884 81.6 92.844z" fill="#85ACB0"/><path d="M90 60c16.569 0 30-13.431 30-30 0-16.569-13.431-30-30-30-16.569 0-30 13.431-30 30 0 16.569 13.431 30 30 30z" fill="#FF444F"/><path d="M85.75 41.875 75 31.125l4.5-4.5 6.25 6.25 14.75-14.75 4.5 4.5-19.25 19.25z" fill="#fff"/></svg>
```

Comment:
Will this icon also look fine in dark theme? If not we need to replace `fill` property with our variable

Reply:
![image](https://user-images.githubusercontent.com/82078941/160543003-727d3913-8f72-4d22-8813-37ad867a591f.png)

Reply:
![image](https://user-images.githubusercontent.com/82078941/160543359-69dea89f-4898-4798-a734-134f61fe9d9a.png)

Reply:
icon looks good in dark theme